app/design/frontend/rwd/default/template/review/form.phtml

```diff
@@ -31,7 +31,7 @@
                             <div class="star-rating">
                             <?php foreach (array_reverse($_rating->getOptions()) as $key=>$_option): ?>
                                 <input required class="star-rating__value" type="radio" id="<?php echo $this->escapeHtml($_rating->getRatingCode()) ?>_<?php echo $_option->getValue() ?>" value="<?php echo $_option->getId() ?>" name="ratings[<?php echo $_rating->getId() ?>]" value="<?php echo $_option->getId() ?>" />
-                                <label for="<?php echo $this->escapeHtml($_rating->getRatingCode()) ?>_<?php echo $_option->getValue() ?>" class="star-rating__label" title="<?php echo $_option->getValue(); ?>"><?php echo $_option->getValue(); ?></label>
+                                <label for="<?php echo $this->escapeHtml($_rating->getRatingCode()) ?>_<?php echo $_option->getValue() ?>" class="star-rating__label" title="<?php echo $this->__($_option->getValue() . ' star'); ?>"><?php echo $_option->getValue(); ?></label>
```

Comment:
genuine question: do you think this is worth adding 5 translations elements to the CSV?

Reply:
It looks like those translations are already in both `Mage_Rating.csv` and `Mage_Review.csv`, but they should be "1 star", "2 stars", etc.

What about adding this method:

```php
class Mage_Rating_Model_Rating_Option extends Mage_Core_Model_Abstract
{
    // ... 

    public function getLabel(): string
    {
        if ($this->getValue() == 1) {
            return Mage::helper('rating')->__('%s star', $this->getValue());
        }
        return Mage::helper('rating')->__('%s stars', $this->getValue());
    }
}
```

And then:

```html
title="<?= $_option->getLabel(); ?>"
```

Reply:
good idea! thanks